### PR TITLE
Enables high priority for processes when benchmarking

### PR DESCRIPTION
**Pull request by zooba**
Haven't _actually_ tested on Linux yet, but it will definitely pass 20 to the function, which the docs say is correct.

### CODE REVIEW

**Comment by zooba** [comment]
I don't have good answers for the failing checks... looks like PyPy isn't installed, and flake8 doesn't know how to count to two...

**Comment by haypo** [comment]
Oh, the pypy job is broken. It should be removed, or fixed.

**Comment by methane** [comment]
Isn't 20 a lowest priority?

**Comment by giampaolo** [comment]
> Isn't 20 a lowest priority?

Oh right.

**Comment by zooba** [comment]
Didn't actually see the comment before I made the fix - I was closing browser windows and re-read the docs :)

File: perf/_runner.py [hunk 1]
```diff
@@ -384,8 +385,13 @@ def _cpu_affinity(self):
                       "isolated CPUs, CPU affinity not available")
                 print("Use Python 3.3 or newer, or install psutil dependency")
 
+    def _process_priority(self):
+        if not set_highest_priority():
+            print("WARNING: unable to increase process priority")
+
     def _worker(self, task):
         self._cpu_affinity()
+        self._process_priority()
```
Comment (haypo) [inline perf/_runner.py]: I'm sorry, but I don't have the bandwidth to review this patch on Linux yet. So I propose you to first make this code Windows specific:

if MS_WINDOWS:
  self._process_priority()

And so always call: proc.nice(psutil.REALTIME_PRIORITY_CLASS)

I'm not sure that call os.nice() on Linux. I don't undertand well the consequences.

**Comment by haypo** [comment]
I removed PyPy from Travis CI: commit 4802fe5add2d26372a91bea0142baf98a25065d1, so if you rebase/merge with master, tests should pass.

**Comment by haypo** [comment]
> Looks good to me in principle but I am not sure how Victor feels about depending on psutil.

For Unix/BSD, os.nice() can be used to not depend on psutil.

For Windows, well, it's an optional dependency, it's fine. If too many users complain, we might provide a ctypes fallback later?

**Comment by haypo** [comment]
@zooba: Please only modify the behaviour on Windows. I prefer to discuss using nice() on Unix later. By the way, it will help you to get a green Travis CI :-)

I'm not sure about using os.nice(). For example, it fails on Travis CI :-)

**Comment by haypo** [comment]
Thank you @zooba for this nice enhancement.

**Comment by serhiy-storchaka** [comment]
nice() can't increase priority unless the process is ran with root privileges.

**Pull request merged**